project.php

```diff
@@ -694,7 +694,13 @@ function do_project_info_table()
                 $blurb = _("Images, Pages Proofread, & Differences");
                 $url2 = "$url&amp;select_by_user";
                 $blurb2 = _("Just my pages");
-                $detail = "<a href='$url'>$blurb</a> &gt;&gt;<a href='$url2'>$blurb2</a>&lt;&lt;";
+                $detail = "<a href='$url'>$blurb</a> | <a href='$url2'>$blurb2</a>";
```

Comment:
We may want to rethink how we're outputting these strings. $blurb1 and $blurb2 point to the same page with different parameters. Presumably we used `>> <<` to draw attention to what most(?) people will want to use -- which is just their pages. Removing that and placing them all in `|`s makes them appear as though they are all equivalent. Perhaps something like: `$blurb1 &middot; <b>$blurb2</b> | $blurb3`? That somewhat mirrors what we do in the navbar.